ldap/servers/plugins/replication/cl5_api.c

```diff
@@ -3030,9 +3029,9 @@ _cl5GenRUVInfo(dbi_val_t *key, dbi_val_t *data, void *ctx)
     ridinfo = _cl5GetRidInfo(dblcictx, rid, PR_TRUE);
     if (ridinfo->new == 1) {
         ridinfo->new = 0;
-        ridinfo->mincsn = csn;
+        ridinfo->mincsn = dblcictx->csn;
     }
-    ridinfo->maxcsn = csn;
+    ridinfo->maxcsn = dblcictx->csn;
```

Comment:
Sorry again same comment. The fix is valid but how did it work before ?
The csn was NULL and not modified. Does that mean the _cl5GenRUVInfo generated ridinfo with NULL min/max csn ?

Reply:
Exactltly